BCB420.2019.ESA.Rproj

```diff
@@ -18,3 +18,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace,vignette
```

Comment:
I don't think you need to add this for roxygenize to work. See if removing it changes anything